cloudnet/src/main/java/de/dytanic/cloudnet/cluster/NodeServer.java

```diff
@@ -0,0 +1,34 @@
+package de.dytanic.cloudnet.cluster;
+
+import de.dytanic.cloudnet.driver.network.cluster.NetworkClusterNode;
+import de.dytanic.cloudnet.driver.network.cluster.NetworkClusterNodeInfoSnapshot;
+import de.dytanic.cloudnet.driver.provider.service.CloudServiceFactory;
+import de.dytanic.cloudnet.driver.provider.service.SpecificCloudServiceProvider;
+import de.dytanic.cloudnet.driver.service.ServiceInfoSnapshot;
+import org.jetbrains.annotations.ApiStatus;
+import org.jetbrains.annotations.NotNull;
+
+public interface NodeServer extends AutoCloseable {
```

Comment:
The close method in AutoCloseable throws an IOException, doesn't it? Maybe just override the close method in this interface so that users don't need to catch the IOException because our implementation can't throw an IOExeption.

Reply:
I'm not sure why but this was already there so i just left to not break any compability